lib/unscopedDisables.mjs

```diff
@@ -0,0 +1,64 @@
+import { RULE_NAME_ALL } from './constants.mjs';
+
+import optionsMatches from './utils/optionsMatches.mjs';
+import reportCommentProblem from './utils/reportCommentProblem.mjs';
+import validateDisableSettings from './validateDisableSettings.mjs';
+
+/** @typedef {import('postcss').Node} Node */
+
+/** @param {import('stylelint').PostcssResult} postcssResult */
+export default function unscopedDisables(postcssResult) {
```

Comment:
[suggestion] For clarity, the `reportUnscopedDisables` name seems better to me (see also `reportDisables()`). How about renaming it like this?

```suggestion
export default function reportUnscopedDisables(postcssResult) {
```

We should have started with `report` for other functions, such as `needlessDisables()` or `invalidScopeDisables()`. I always feel it's hard to find such reporting functions. 😅

Reply:
> We should have started with `report` for other functions, such as `needlessDisables()` or `invalidScopeDisables()`.

I wanted to follow the previous convention.
I don't mind the rename.

Reply:
I still want to follow a better convention for a new code so that we can easily refactor the old code later. Since `reportDisables()` already exists, I think `reportUnscopedDisables()` also makes sense and isn't weird.

Reply:
I already did the rename.